packages/ui/ViewManager/tests/View-specs.js

```diff
@@ -132,7 +141,8 @@ describe('View', () => {
 				done();
 			});
 
-			ref.current.componentWillStay(spy);
+			act(() => ref.current.componentWillStay(spy));
+			jest.runAllTimers();
```

Comment:
Please check whether this is  necessary or not.

Reply:
maybe..

```suggestion
			done();
```

Reply:
Since setTimeout is used in the Mock Arranger function, we need to advance the timer's time with jest.runAllTimers() .